dissect/target/plugins/browsers/chrome.py

```diff
@@ -1,116 +1,32 @@
-from dissect.sql import sqlite3
-from dissect.sql.exceptions import Error as SQLError
-from dissect.util.ts import webkittimestamp
-
-from dissect.target.exceptions import FileNotFoundError, UnsupportedPluginError
 from dissect.target.helpers.descriptor_extensions import UserRecordDescriptorExtension
 from dissect.target.helpers.record import create_extended_descriptor
 from dissect.target.plugin import Plugin, export
-from dissect.target.plugins.browsers.browser import (
-    GENERIC_HISTORY_RECORD_FIELDS,
-    try_idna,
-)
-
-ChromeBrowserHistoryRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
-    "browser/chrome/history", GENERIC_HISTORY_RECORD_FIELDS
-)
+from dissect.target.plugins.browsers.browser import GENERIC_HISTORY_RECORD_FIELDS
+from dissect.target.plugins.browsers.chromium import ChromiumMixin
 
 
-class ChromePlugin(Plugin):
+class ChromePlugin(ChromiumMixin, Plugin):
     """Chrome browser plugin."""
 
+    ChromeBrowserHistoryRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
+        "browser/chrome/history", GENERIC_HISTORY_RECORD_FIELDS
+    )
     __namespace__ = "chrome"
 
     DIRS = [
+        # Windows
         "AppData/Local/Google/Chrome/User Data/Default",
         "AppData/Local/Google/Chrome/continuousUpdates/User Data/Default",
         "Local Settings/Application Data/Google/Chrome/User Data/Default",
         "AppData/local/Google/Chromium/User Data/Default",
+        # Linux
         ".config/google-chrome/Default",
-        "snap/chromium/common/chromium/Default",
+        # Macos
+        "Library/Application Support/Google/Chrome/Default",
     ]
+    HISTORY_RECORD = ChromeBrowserHistoryRecord
```

Comment:
Maybe do something like this?
```suggestion
    HISTORY_RECORD = ChromeBrowserHistoryRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
        "browser/chrome/history", GENERIC_HISTORY_RECORD_FIELDS
    )
```
Instead of defining them first separately in the class?